### PR TITLE
Armstrong numbers add test template

### PR DESCRIPTION
**Pull request by K4cePhoenix**
This PR adds a template for armstrong numbers #1927 

However I'm having an issue generating the tests. The generator script doesn't find the template. 
Running `bin/generate_tests.py -v armstrong-numbers` gives me the following console output
`DEBUG:generator:armstrong-numbers\.meta\template.j2`
`INFO:generator:armstrong-numbers: no template found; skipping`

### CODE REVIEW

**Comment by cmccandless** [comment]
```
$ bin/generate_tests.py -v armstrong-numbers
INFO:generator:armstrong-numbers: attempting render
armstrong-numbers generated at exercises/armstrong-numbers/armstrong_numbers_test.py
$ git diff
diff --git a/exercises/armstrong-numbers/armstrong_numbers_test.py b/exercises/armstrong-numbers/armstrong_numbers_test.py
index 46b7fc5..98fb58b 100644
--- a/exercises/armstrong-numbers/armstrong_numbers_test.py
+++ b/exercises/armstrong-numbers/armstrong_numbers_test.py
@@ -2,38 +2,55 @@ import unittest
 
 from armstrong_numbers import is_armstrong_number
 
-
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
 
-class ArmstrongNumbersTest(unittest.TestCase):
 
+class ArmstrongNumbersTest(unittest.TestCase):
     def test_zero_is_an_armstrong_number(self):
-        self.assertIs(is_armstrong_number(0), True)
+        input = 0
+        expected = True
+        self.assertEqual(is_armstrong_number(input), expected)
 
     def test_single_digit_numbers_are_armstrong_numbers(self):
```

Placing your template in exercises/armstrong-numbers/.meta/template.j2 works for me. What directory are you running generate_tests.py from?

**Comment by cmccandless** [comment]
The Travis build had no trouble finding the template; it's failing because the generated output doesn't match the current tests file.

**Comment by K4cePhoenix** [comment]
> What directory are you running generate_tests.py from?

I'm running generate_tests.py from the repository's root folder, which is "E:/workspaces/github/python" on my machine.

**Comment by cmccandless** [comment]
That is strange... can you try deleting `exercises/two-fer/two_fer_test.py` and running `bin/generate_tests.py -v two-fer`? Does it re-generate `two_fer_test.py`?

**Comment by K4cePhoenix** [comment]
It results in the following
> E:\workspaces\github\python>bin/generate_tests.py -v two-fer
DEBUG:generator:two-fer\.meta\template.j2
INFO:generator:two-fer: no template found; skipping

**Comment by cmccandless** [comment]
> DEBUG:generator:two-fer.meta\template.j2

That's not a valid path. The previous output you gave was formatted correctly; was this hand-typed?

**Comment by K4cePhoenix** [comment]
>>DEBUG:generator:two-fer.meta\template.j2

>That's not a valid path. The previous output you gave was formatted correctly; was this hand-typed?

Not sure how I lost that backslash, but the actual output was `DEBUG:generator:two-fer\.meta\template.j2`.

I fixed my issue by installing and setting up WSL and using it to generate the `armstrong_numbers_test.py` test file.

File: exercises/armstrong-numbers/.meta/template.j2 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        input = {{ case["input"]["number"] }}
+        expected = {{ case["expected"] }}
+        self.assertEqual({{ case["property"] | to_snake }}(input), expected)
```
Comment (cmccandless) [inline exercises/armstrong-numbers/.meta/template.j2]: I would suggest putting everything in the `assertEqual` call:
```
self.assertEqual(
    {{ case["property"] |to_snake }}({{ case ["input"]["number"] }}), 
    {{ case["expected"] }}
)
```

The generated result will produce a smaller diff from the original.

Comment (K4cePhoenix) [reply exercises/armstrong-numbers/.meta/template.j2]: Done :)

**Pull request merged**